src/dstore/dstore_base.c

```diff
@@ -289,7 +244,8 @@ static int dstore_io_op_init_and_submit(struct dstore_obj *obj,
 	dassert(dstore_io_vec_invariant(bvec));
 	/* Only WRITE/READ is supported so far */
```

Comment:
Please update the comment for free/truncate/unmap operation

Reply:
Done. Code Merged. Rand UT and CTHON again.